### ISSUE TITLE
Cannot delete a User with Angular client

### ISSUE THREAD

**Issue by chrisdns** [jhipster/generator-jhipster#7610]
##### **Overview of the issue**

I cannot delete an User with `user-management` menu.

##### **Reproduce the error**

Create a regular Angular app and try to delete an user with the `user-management` menu.

##### **JHipster Version(s)**

Current master

##### **JHipster Version(s)**

```
test-angular@0.0.0 /home/alexandregc/repos/testAngular
└── generator-jhipster@5.0.0-beta.1  -> /home/alexandregc/repos/generator-jhipster

```

##### **JHipster configuration, a `.yo-rc.json` file generated in the root folder**

<details>
<summary>.yo-rc.json file</summary>
<pre>
{
  "generator-jhipster": {
    "promptValues": {
      "packageName": "com.mycompany.myapp",
      "nativeLanguage": "en"
    },
    "jhipsterVersion": "5.0.0-beta.1",
    "baseName": "testAngular",
    "packageName": "com.mycompany.myapp",
    "packageFolder": "com/mycompany/myapp",
    "serverPort": "8080",
    "authenticationType": "jwt",
    "cacheProvider": "ehcache",
    "enableHibernateCache": true,
    "websocket": false,
    "databaseType": "sql",
    "devDatabaseType": "h2Disk",
    "prodDatabaseType": "mysql",
    "searchEngine": false,
    "messageBroker": false,
    "serviceDiscoveryType": false,
    "buildTool": "maven",
    "enableSwaggerCodegen": false,
    "jwtSecretKey": "replaced-by-jhipster-info",
    "clientFramework": "angularX",
    "useSass": false,
    "clientPackageManager": "yarn",
    "applicationType": "monolith",
    "testFrameworks": [],
    "jhiPrefix": "jhi",
    "enableTranslation": true,
    "nativeLanguage": "en",
    "languages": [
      "en"
    ]
  }
}
</pre>
</details>

##### **JDL for the Entity configuration(s) `entityName.json` files generated in the `.jhipster` directory**

<details>
<summary>JDL entity definitions</summary>

<pre>

</pre>
</details>

##### **Environment and Tools**

java version "1.8.0_171"
Java(TM) SE Runtime Environment (build 1.8.0_171-b11)
Java HotSpot(TM) 64-Bit Server VM (build 25.171-b11, mixed mode)

git version 2.17.0

node: v8.11.1

npm: 5.6.0

yeoman: 2.0.1

yarn: 1.6.0

Docker version 17.12.1-ce, build 7390fc6

docker-compose version 1.17.1, build unknown

##### **Browsers and Operating System**

Firefox, Ubuntu 18.04

**Comment by ruddell** [comment]
This is addressed by #7406

**Comment by pascalgrimaud** [comment]
Closing this, as https://github.com/jhipster/generator-jhipster/pull/7406 is merged
